source/DataIngestionApi.ts

```diff
@@ -72,6 +73,15 @@ export class DataIngestionApi {
         return this.invoke<ITenantSettingsApiModel>(params, pathTemplate, method, this.additionalParams, body);
     }
 
+    public notifyFailure() {
```

Comment:
I think this would just need to accept either an object or a few params so that the lambda can have the necessary params to properly raise the event. I think these would all be part of the body object

- tenantName
- tenantID
- error